.gitlab-ci.yml

```diff
@@ -1,12 +1,35 @@
 stages:
+  - vulnerabilities
   - test
 
-before_script:
-  - curl -sS https://getcomposer.org/installer | php
-  - php composer.phar install --no-scripts --quiet --ignore-platform-reqs
-  - cp app/config/parameters.yml.dist app/config/parameters.yml
+######################################
+# Vulnerabilities stage related jobs #
+######################################
+check php vendors:
+  stage: vulnerabilities
+  image: sumocoders/framework-php72
+  before_script:
+    - touch .env
```

Comment:
waarvoor wordt dat bestand gebruikt?

Reply:
Fixed in 3eb3050366090c3266471e1accb049826c52bcb6